integrationtests/src/test/java/io/kroxylicious/proxy/multitenant/MultiTenantIT.java

```diff
@@ -190,30 +194,37 @@ public void tenantTopicIsolation(KafkaCluster cluster) throws Exception {
         }
     }
 
-    @Test
-    public void tenantConsumeWithGroup(KafkaCluster cluster) throws Exception {
+    private enum ConsumerStyle {
+        ASSIGN,
+        SUBSCRIBE
+    }
+
+    @ParameterizedTest
+    @EnumSource
```

Comment:
Is there any advantage to using `@ParameterizedTest` with `@EnumSource` over having two test cases which explicitly name what they are demonstrating and pass the enum directly?

Reply:
the test is essentially the same, the difference being the consumer gesture (subscription vs assign).    If we were to give them specific names we'd probably end up with test methods like `tenantGroupIsolation_consumerUsesSubscription` and `tenantGroupIsolation_consumerUsesAssignment`.   I have a slight preference for the compactness of the enumsource, but don't feel strongly, so will happily change.

FWIW, I think this might be the only case where we need to do something like this.

Reply:
We are going to merge this PR, but happy to change this in a future PR if the comments above don't answer the concern!